torchq/mesolve.py

```diff
@@ -0,0 +1,65 @@
+from typing import Any, Dict, List, Optional
+
+import numpy as np
+import torch
+import torchdiffeq
+
+
+def mesolve(
+    H: torch.Tensor,
+    Ls: List[torch.Tensor],
+    rho0: torch.Tensor,
+    tlist: np.ndarray,
+    solver: str = 'rk4',
+    options: Optional[Dict[str, Any]] = None,
+) -> torch.Tensor:
+    if options is None:
+        options = {}
+
+    if solver == 'rk4':
+        if not 'step_size' in options:
+            options['step_size'] = 0.001
+        return _mesolve_torchdiffeq(H, Ls, rho0, tlist, options)
+
+    raise ValueError('invalid solver (supported: \'rk4\')')
```

Comment:
Why do directly sending the `solver` parameter to `torchdiffeq.odeint` `method` parameter?

Reply:
I just didn't try the other solvers, but I agree we could do as you suggest 👍